maxim/logger/livekit/agent_session.py

```
@@ -0,0 +1,172 @@
+        f"[{self.__class__.__name__}] Agent state updated; new_state={new_state}"
+    )
+    trace = get_session_store().get_current_trace_for_session(id(self))
+    if trace is not None:
+        trace.event(str(uuid.uuid4()), "agent_state_updated", {"new_state": new_state})
+
+
+def intercept_generate_reply(self, *args, **kwargs):
+    """
+    This function is called when the agent generates a reply.
+    """
+    instructions = kwargs.get("instructions", None)
```

Ruff (0.11.9)

Use `kwargs.get("instructions")` instead of `kwargs.get("instructions", None)`

Replace `kwargs.get("instructions", None)` with `kwargs.get("instructions")`

(SIM910)